src/TabbedWebView.vala

```diff
@@ -61,6 +69,40 @@ public class Captive.TabbedWebView : WebKit.WebView {
         });
     }
 
+    public string security_to_string () {
```

Comment:
It would be cleaner to have these two methods could be member methods of the `Security`  enum

Reply:
Yeah I was gonna do that at first but the problem is they don't have access to the uri because that's in each instance

Reply:
Oh I could do that for icons though!

Reply:
> Yeah I was gonna do that at first but the problem is they don't have access to the uri because that's in each instance

Ah yeah, my bad. Hadn't noticed you'd used the URL in there!

Reply:
Fix the icon though! Thanks for the review :)